common/rya.api/src/main/java/org/apache/rya/api/instance/RyaDetailsToConfiguration.java

```diff
@@ -53,14 +53,16 @@ public static void addRyaDetailsToConfiguration(final RyaDetails details, final
         checkAndSet(conf, ConfigurationFields.USE_FREETEXT, details.getFreeTextIndexDetails().isEnabled());
       //RYA-215        checkAndSet(conf, ConfigurationFields.USE_GEO, details.getGeoIndexDetails().isEnabled());
         checkAndSet(conf, ConfigurationFields.USE_TEMPORAL, details.getTemporalIndexDetails().isEnabled());
-        PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
-		if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
-			checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
-			conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
-			conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
-		} else {
-			checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, false);
-		}
+        final PCJIndexDetails pcjDetails = details.getPCJIndexDetails();
+        if (pcjDetails.isEnabled() && pcjDetails.getFluoDetails().isPresent()) {
+            checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, true);
+            conf.set(ConfigurationFields.FLUO_APP_NAME, pcjDetails.getFluoDetails().get().getUpdateAppName());
+            conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "FLUO");
+            conf.set(ConfigurationFields.PCJ_STORAGE_TYPE, "ACCUMULO");
+        } else {
+            checkAndSet(conf, ConfigurationFields.USE_PCJ_UPDATER, false);
+            conf.set(ConfigurationFields.PCJ_UPDATER_TYPE, "NO_UPDATE");
```

Comment:
I completely agree with you on not using the String literals.  Unfortunately `PrecomputedJoinUpdaterType` is in rya.indexing and `RyaDetailsToConfiguration` is in rya.api, so I can't access that type.  I believe (could be wrong) that Caleb had the action item to create (or reference) a JIRA that would enhance the `PCJIndexDetails` class to have a `PCJIndexDetails.getUpdaterType()` and `PCJIndexDetails.getStorageType()` methods, or perhaps we might go in some other direction with the implementation.